### PR TITLE
[EmbeddingAPI] Add a usecase to create a xwalkview as a scrollview child

### PR DESCRIPTION
**Pull request by jacky-young**
-Add a usecase to create a xwalkview as a scrollview child
-Cover the XWalkActivity and XWalkInitializer two ways
-Modify README source code file path

Impacted tests(approved): new 2, update 0, delete 0
Unit test platform: [Android]
Unit test result summary: pass 0, fail 2, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-4987

### CODE REVIEW

**Comment by jacky-young** [comment]
@Shao-Feng please help to review it.

File: usecase/usecase-embedding-android-tests/embeddingapi/res/layout/activity_xwalk_view_with_scrollview_parent.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Shao-Feng) [inline usecase/usecase-embedding-android-tests/embeddingapi/res/layout/activity_xwalk_view_with_scrollview_parent.xml]: pls define the scroll view in a layout.
and add textview for test description

**Comment by Shao-Feng** [comment]
the test description should be updated

**Comment by jacky-young** [comment]
@Shao-Feng please review again.

**Comment by Shao-Feng** [comment]
LGTM

**Comment by AAverin** [comment]
Provided test fails. XWalkView is not displayed within ScrollView
Used: Android Emulator v21

https://github.com/crosswalk-project/crosswalk-test-suite/issues/3409

**Comment by zqzhang** [comment]
> Unit test result summary: pass 0, fail 2, block 0

@AAverin the unit test results told us that the tests are failed, by the way. That means the tests shall be improved some time later aligning with updated implementation. Also the tests themselves may be buggy.

**Comment by jacky-young** [comment]
Hi @zqzhang This is the original bug [XWALK-4876](https://crosswalk-project.org/jira/browse/XWALK-4876), it's still unresolved. My pr is just a demonstration or usecase to show the problem, And after bug fixed, my case should work fine.

**Pull request merged**